python/mqt/syrec/quantum_circuit_simulation_dialog.py

```
@@ -0,0 +1,1241 @@
+            message_box_parent=self,
+            message_box_title="Synthesized SyReC program contained line or block comments!",
+            message_box_content="SyReC programs containing line or block comments cannot be serialized to JSON since this would generate invalid JSON. Export to JSON functionality is disabled.",
+            is_cancellable=False,
+            log_contents=False,
+        )
+
+    def initialize_simulation_runs_tab_widget(
+        self,
+        shared_simulation_runs_model: QtSimulationRunModel,
+        tab_widget_object_name: str,
+        create_load_from_file_controls: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)